recipes/cis/meta.yaml

```diff
@@ -0,0 +1,60 @@
+package:
+  name: cis
+  version: "1.3.2"
+
+source:
+  #path: ../../cis
+  git_url: https://github.com/cedadev/cis.git
+  git_tag: 1.3.2
```

Comment:
We prefer tarballs to git tags because they are a more absolute point in time.  People can change tags (though it is certainly frowned on), and we would not notice.  With tarballs, we would definitely notice.